### DISCUSSION TITLE
2 Questions: Browser and Advanced

### DISCUSSION THREAD

**Discussion by Bondarz** [keepassxreboot/keepassxc#8064] [Q&A]
Hello,

KeePassXC - 2.7.1
KeePassXC-Browser - 1.7.12
Operating system: Win64
Browser: Mozilla Firefox 100.0 

Browser: example Paypal. Here I have several logins at KeePass, I am always displayed only 2 of 3 for Autofill. (Keepass browser access request)
-> How do I get (again) a third entry deposited?

Can I store multiple passwords in the main view, or only one password? 

Example: I use more than one password for my Raspberry Pi. But I do not want to create multiple entries. 
-> Can I add more fields for passwords here, or do I have to solve this via "Advanced" and "Additional properties"?

Thanks

**Comment by droidmonkey** [comment]
Please read the user guide, there is a lot of information in there about entry management.
